functorch/csrc/BatchRulesReduceOps.cpp

```diff
@@ -255,6 +255,42 @@ std::tuple<Tensor,optional<int64_t>> _log_softmax_backward_batch_rule(
   return std::make_tuple(at::_log_softmax_backward_data(grad_output_, output_, dim, input_dtype), 0);
 }
 
+std::tuple<Tensor, optional<int64_t>, Tensor, optional<int64_t>> aminmax_batching_rule(
+    const Tensor &self, optional<int64_t> self_bdim, optional<int64_t> dim, bool keep_dim)
+{
+  Tensor min, max;
+  if (!self_bdim.has_value())
```

Comment:
I think this case never happens now (if no input tensors have batch dims we don't dispatch to the batch rule)